.github/workflows/build_docker.yaml

```diff
@@ -0,0 +1,52 @@
+name: Docker build
+on:
+  pull_request:
+    branches:
+    - main
+  workflow_dispatch: {}
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'mosaicml'
+    strategy:
+      fail-fast: false
```

Comment:
Is there a reason we're overriding the default behavior here? ([fail-fast docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast))

Reply:
I am not sure... this is from @bandish-shah , will let him comment

Reply:
That wasn't intentional probably just a copy/paste artifact from whatever flow I originally based this on.  You can enable it if you want, doesn't really matter right now since the build is so quick and you're only building 2 images.

Reply:
I will revert to default as we are doing this on PRs